angular/src/app/landing/landing.component.ts

```diff
@@ -304,10 +298,30 @@ updateMenu(){
           this.onError(" There is an error");
           // throw new ErrorComponent(this.route);
        });
-    
   }
 
-  ngAfterViewInit(){}
+  gotoSelection(){
+    this.router.events.subscribe(s => {
+      if (s instanceof NavigationEnd) {
+        const tree = this.router.parseUrl(this.router.url);
+        if (tree.fragment) {
+          const element = document.querySelector("#" + tree.fragment);
+         console.log("Element"+tree.fragment);
+          if (element) { 
+            //element.scrollIntoView(true); 
+            element.scrollIntoView({behavior: "instant", block: "start", inline: "start"});
+           }
+        }
+      }
+    });
+  }
+
+  ngAfterViewInit(){
+    this.gotoSelection();
+    if (this.record != null && isPlatformBrowser(this.platformId) )  {
+      window.history.replaceState( {} , 'pdr/od/id/', '/od/id/'+this.searchValue );
```

Comment:
It looks like there might be some confusion about meaning of the 2nd argument to `replaceState()`.  According to the [Mozilla developer documentation](https://developer.mozilla.org/en-US/docs/Web/API/History_API), the second argument is defined as:
> title — Firefox currently ignores this parameter, although it may use it in the future. Passing the empty string here should be safe against future changes to the method. Alternatively, you could pass a short title for the state to which you're moving.

To future-proof and otherwise avoid propagating the confusion, I would recommend changing the 2nd argument to an empty string.